bin/git-overwritten

```diff
@@ -116,7 +119,7 @@ git diff ${diff_type-"${base}...${head}"} --diff-filter=DM --no-prefix -w -U0 |
   fi
 
   while IFS=$'\t' read time sha author msg; do
-    date -r $time "+%Y-%m-%d" | tr -d $'\n'
+    date "${DATEARG}$time" "+%Y-%m-%d" | tr -d $'\n'
```

Comment:
Sure. That was leftover from an attempt to do this with `${GNU:+-d@}` type syntax and really isn't even necessary, but I guess with two variable smashed together reads better with braces, so both it is.